.github/workflows/ci.yml

```diff
@@ -190,6 +190,25 @@ jobs:
       - name: Check types
         run: pnpm types:check
 
+  knip:
+    name: Run knip checks
```

Comment:
Knip requires package to be build in order to work, so I created a new CI job.
([Right now it takes ~3s](https://github.com/scalar/scalar/actions/runs/19007001657/job/54282448316?pr=7233) but this time might be increased while adding new packages)